ibis/backends/mysql/__init__.py

```diff
@@ -86,3 +91,47 @@ def connect(
             driver=driver,
         )
         return self.client
+
+    @contextlib.contextmanager
+    def begin(self):
+        with super().begin() as bind:
+            previous_timezone = bind.execute(
+                'SELECT @@session.time_zone'
+            ).scalar()
+            try:
+                bind.execute("SET @@session.time_zone = 'UTC'")
+            except Exception as e:
+                warnings.warn(f"Couldn't set mysql timezone: {str(e)}")
```

Comment:
I think this warning is pretty useless, but we can evaluate that again later.

Reply:
Also moved this from Client without changes, were this was implemented.